src/N3Parser.js

```diff
@@ -614,6 +635,17 @@ export default class N3Parser {
     case ',':
       next = this._readObject;
       break;
+    case '{|':
+      if (!this._supportsRDFStar)
+        return this._error('Unexpected RDF* syntax', token);
```

Comment:
Do you really mean the deprecated `RDF*`, or should this name the current `RDF-star`?
```suggestion
        return this._error('Unexpected RDF* syntax', token);
```